integrationtests/proxy/proxy_test.go

```diff
@@ -38,44 +38,45 @@ var _ = Describe("QUIC Proxy", func() {
 
 	Context("Proxy setup and teardown", func() {
 		It("sets up the UDPProxy", func() {
-			proxy, err := NewQuicProxy("localhost:13370", Opts{RemoteAddr: serverAddr})
+			proxy, err := NewQuicProxy("localhost:0", Opts{RemoteAddr: serverAddr})
 			Expect(err).ToNot(HaveOccurred())
 			Expect(proxy.clientDict).To(HaveLen(0))
 
-			// check that port 13370 is in use
-			addr, err := net.ResolveUDPAddr("udp", "localhost:13370")
+			// check that the proxy port is in use
+			addr, err := net.ResolveUDPAddr("udp", "localhost:"+strconv.Itoa(proxy.LocalPort()))
 			Expect(err).ToNot(HaveOccurred())
 			_, err = net.ListenUDP("udp", addr)
-			Expect(err).To(MatchError("listen udp 127.0.0.1:13370: bind: address already in use"))
+			Expect(err).To(MatchError("listen udp 127.0.0.1:" + strconv.Itoa(proxy.LocalPort()) + ": bind: address already in use"))
 
-			err = proxy.Close() // stopping is tested in the next test
-			Expect(err).ToNot(HaveOccurred())
+			Expect(proxy.Close()).To(Succeed()) // stopping is tested in the next test
 		})
 
 		It("stops the UDPProxy", func() {
-			proxy, err := NewQuicProxy("localhost:13371", Opts{RemoteAddr: serverAddr})
+			proxy, err := NewQuicProxy("localhost:0", Opts{RemoteAddr: serverAddr})
 			Expect(err).ToNot(HaveOccurred())
+			port := proxy.LocalPort()
 			err = proxy.Close()
 			Expect(err).ToNot(HaveOccurred())
 
-			// check that port 13371 is not in use anymore
-			addr, err := net.ResolveUDPAddr("udp", "localhost:13371")
-			Expect(err).ToNot(HaveOccurred())
-			ln, err := net.ListenUDP("udp", addr)
-			Expect(err).ToNot(HaveOccurred())
-			err = ln.Close()
+			// check that the proxy port is not in use anymore
+			addr, err := net.ResolveUDPAddr("udp", "localhost:"+strconv.Itoa(port))
 			Expect(err).ToNot(HaveOccurred())
+			// sometimes it takes a while for the OS to free the port
```

Comment:
That shouldn't be hard, since this has nothing to do with QUIC at all. However, I'm not sure who's at fault here, I've only seen this bug on Travis.